core/trigProducers.fcl

```diff
@@ -771,9 +772,9 @@ TrigCstProducers : {
       t0offset: 0
    }
    CstSimpleTimeCluster: {
+      @table::SimpleTimeCluster
```

Comment:
- why can Ed just integrate the proposed change? The namespace is already there in the prolog file included, so no changes are necessary to the Offline code
- I wanted to make sure that the trigger executable runs without errors. @edcallaghan you need to switch to the Off spill menu in the triggerTest.fcl file (at lines 13 and 14)

Reply:
@gianipez changing the menu to offspill does not change the definition of `TTtimeClusterFinder`, which is the module with incomplete configuration, and thus the test job still fails for that reason.

Reply:
Hi @edcallaghan , I finally was capable to run the triggerTest.fcl using the head of the Offline and mu2e_trig_config:
`mu2e -c mu2e_trig_config/test/triggerTest.fcl -S sam_NoPrimaryMix1BBUntriggered.MDC2020r_best_v1_0.txt -n 100` and faced no errors. Could you please try to run it as well? and post here the error log in case of failure?